File: engine/engine-model/src/main/scala/tools/packet/spawn/RemoveDoor.scala
package tools.packet.spawn

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class RemoveDoor(private var _ownerId: Int, private var _town: Boolean) extends PacketInput {
  def ownerId: Int = _ownerId

  def town: Boolean = _town

  override def opcode(): SendOpcode = SendOpcode.REMOVE_DOOR
}

File: engine/engine-model/src/main/java/tools/packet/spawn/RemoveDoor.java
package tools.packet.spawn;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record RemoveDoor(Integer ownerId, Boolean town) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.REMOVE_DOOR;
   }
}